services/src/main/resources/org/keycloak/protocol/oidc/endpoints/login-status-iframe.html

```diff
@@ -55,7 +55,12 @@
           init = { clientId, origin };
         }
 
-        const cookie = getSessionCookie();
+        var cookie = getSessionCookie();
```

Comment:
Note that we read cookies in JavaScript in multiple places, including the session cookie, those will likely have to be adjusted as well. If you search for `document.cookie` you will find a couple of such occurrences.